tasks/task109_SMSspamcollection_SpamSMSdetection.json

```diff
@@ -0,0 +1,11545 @@
+{
+    "Contributors": [
+        "Savan Doshi"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Source": [
+        "SMS Spam Collection v.1 (https://www.dt.fee.unicamp.br/~tiago/smsspamcollection/)"
+    ],
+    "Definition": "In this task, you're given Text messages which can be classified between two classes Spam or Ham messages based on the content of the text message. Spam messages are messages which contain text which is irrelevant to the receiver and the message can be equally applicable to many other potential recipients.",

```

Comment:
It's unclear what is "Ham".  First, define "Spam": *Label a given text message "Spam", if ... . Otherwise, label it as "Ham"*

Reply:
I again updated definition please verify.